contributing/code/standards.rst

```diff
@@ -172,6 +172,10 @@ Documentation
 
 * Add PHPDoc blocks for all classes, methods, and functions;
 
+* Group annotations together so that annotations of the same type immediately
```

Comment:
OK. I finally understood it. So maybe we could say something like "group annotations by name"?